src/renderer/views/Settings/Settings.tsx

```diff
@@ -30,70 +32,78 @@ const Settings = () => {
-              collaborators: [...projectSettings.collaborators, collaborator],
-            })
-          }
-          onDelete={(id) =>
-            handleChange({
-              collaborators: projectSettings.collaborators.filter(
-                (collaborator) => !(collaborator.id === id)
-              ),
-            })
-          }
-        />
-      </SeparatedSection>
-      <SeparatedSection pb={3}>
-        <SnippetsManager state={projectSettings} onSubmit={handleChange} />
-      </SeparatedSection>
-      <SeparatedSection pb={3}>
-        <Button
-          variant='contained'
-          color='green'
-          sx={{ m: 0 }}
-          isMajor
-          fullWidth
-          disabled={!canSubmit}
-          onClick={() => {
-            updateConfig(project.dirPath, changes);
-            dispatch(updatePublication({ ...project, ...changes }));
-            dispatch(
-              sendNotification({
-                title: t('ProjectSettings.notification.title'),
-                message: t('ProjectSettings.notification.message'),
-                type: 'success',
-                autoDismiss: true,
+    <>
+      <ViewContent>
+        <Typography variant='h1' mb={4}>
+          {projectSettings.name}
+        </Typography>
+        <SeparatedSection pb={3}>
+          <ProjectDetailsInput
+            state={projectSettings}
+            onSubmit={handleChange}
+            onValidationStateChange={setCanSubmit}
+          />
+        </SeparatedSection>
+        <SeparatedSection pb={3}>
+          <TagsManager state={projectSettings} onSubmit={handleChange} />
+        </SeparatedSection>
+        <SeparatedSection pb={3}>
+          <CollaboratorsPicker
+            state={projectSettings}
+            compact
+            onAdd={(collaborator) =>
+              handleChange({
+                collaborators: [...projectSettings.collaborators, collaborator],
               })
-            );
-          }}
-        >
-          {t('common.save')}
-        </Button>
-      </SeparatedSection>
-    </ViewContent>
+            }
+            onDelete={(id) =>
+              handleChange({
+                collaborators: projectSettings.collaborators.filter(
+                  (collaborator) => !(collaborator.id === id)
+                ),
+              })
+            }
+          />
+        </SeparatedSection>
+        <SeparatedSection pb={3}>
+          <SnippetsManager state={projectSettings} onSubmit={handleChange} />
+        </SeparatedSection>
+        <SeparatedSection pb={3}>
+          <Button
+            variant='contained'
+            color='green'
+            sx={{ m: 0 }}
+            isMajor
+            fullWidth
+            disabled={!canSubmit}
+            onClick={async () => {
+              const id = uuidv4();
+              setLoaderID(id);
+              await updateConfig(project.dirPath, changes);
+              await gitCommit('Config update\n\n[publAB automatic commit]');
```

Comment:
@LostInStatic some casing issues creeped in to the commit message

Reply:
Funny, I was SURE this is how we style it at that moment